content/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -124,17 +154,176 @@ <h3>Service 和 Label</h3>
+				-->
         <p> 标签(Label)可以在创建时或之后附加到对象上。他们可以随时被修改。现在使用 Service 发布我们的应用程序并添加一些 Label 。</p>
 			</div>
 		</div>
 		<br>
 		<div class="row">
 			<div class="col-md-12">
-<!--        <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/expose/expose-interactive/" role="button">Start Interactive Tutorial<span class="btn__next">›</span></a>-->
-        <a class="btn btn-lg btn-success" href="/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-interactive/" role="button">开始交互式教程<span class="btn__next">›</span></a>
+				<!-- <h3>Create a new Service</h3> -->
+				<h3>创建新 Service</h3>
+				<!--
+				<p>Let’s verify that our application is running. We’ll use the <code>kubectl get</code> command and look for existing Pods:</p>
+				-->
+				<p>让我们来验证我们的应用程序正在运行。我们将使用 <code>kubectl get</code> 命令并查找现有的 Pod ：</p>
+				<p><code><b>kubectl get pods</b></code></p>
+				<!--
+				<p>If no Pods are running then it means the objects from the previous tutorials were cleaned up. In this case, go back and recreate the deployment from the <a href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro#deploy-an-app">Using kubectl to create a Deployment</a> tutorial.
+          Please wait a couple of seconds and list the Pods again. You can continue once you see the one Pod running.</p>
+				<p>Next, let’s list the current Services from our cluster:</p>
+				-->
+				<p>如果没有 Pod 正在运行，则意味着之前教程中的对象已被清理。 在这种情况下，请返回并从 <a href="/zh-cn/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro#deploy-an-app">使用 kubectl 创建 Deployment</a> 教程重新创建 deployment。
+           请等待几秒钟，然后再次列出 Pod。 一旦看到一个 Pod 正在运行，您就可以继续。</p>
+				<p>接下来，让我们从集群中列出当前的 Service ：</p>
+				<p><code><b>kubectl get services</b></code></p>
+				<!--
+				<p>We have a Service called <tt>kubernetes</tt> that is created by default when minikube starts the cluster.
+				To create a new service and expose it to external traffic we'll use the expose command with NodePort as parameter.</p>
+				-->
+				<p>我们有一个名为 <tt>kubernetes</tt> 的 Service ，它在 minikube 启动集群时默认创建.
+				要创建一个新的 Service 然后暴露给外部流量，我们将使用 <code>expose</code> 命令，并将 NodePort 作为参数。</p>
+				<p><code><b>kubectl expose deployment/kubernetes-bootcamp --type="NodePort" --port 8080</b></code></p>
+				<!--
+				<p>Let's run again the <code>get services</code> subcommand:</p>
+				-->
+				<p>让我们再次运行 <code>get services</code> 子命令：</p>
+				<p><code><b>kubectl get services</b></code></p>
+				<!--
+				<p>We have now a running Service called kubernetes-bootcamp. Here we see that the Service received a unique cluster-IP, an internal port and an external-IP (the IP of the Node).</p>
+				<p>To find out what port was opened externally (for the <tt>type: NodePort</tt> Service) we’ll run the <code>describe service</code> subcommand:</p>
+				-->
+				<p>我们现在有一个运行中的 Service 名为 kubernetes-bootcamp 。这里我们看到 Service 收到了一个唯一的 cluster-IP ，一个内部端口和一个 external-IP (Node 的 IP)。</p>
```

Comment:
```suggestion
				<p>我们现在有一个运行中的 Service 名为 kubernetes-bootcamp。
				这里我们看到 Service 收到了一个唯一的 cluster-IP、一个内部端口和一个
				external-IP（Node 的 IP）。</p>
```
长句适当在标点、中英、数字之间折行，方便 review，也会有美感